integration-tests/integration-test.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+integration_test_root="$(readlink -f "$(dirname "${0}")")"
+error_prone_support_root="${integration_test_root}/.."
+repos_root="${integration_test_root}/.repos"
+
+if [ "${#}" -ne 10 ]; then
+  >&2 echo "Usage $(basename "${0}") <test-name> <project> <repository> <revision> <build-flags> <patch-flags> <validation-ep-flags> <validation-mvn-flags> <do-sync> <report-directory>"
```

Comment:
For this, we should take a look at the other `.sh` files in this repository. I believe we have a slightly different naming situation here (which is closer to the official conventions I believe). 

So I _think_ it should be `[TestName] [Project] ....`